jenkins_pipelines/environments/manager-5.1-qe-build-validation-NUE

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env groovy
```

Comment:
If we start changing names... I wonder if we should call it PRG instead.
@Bischoff what do you think?

Reply:
From Eric:
>No hard feelings.
I do not plan to start the infra DNS part any time soon, because it's a lot of work, because it needs cooperation from external infra, and because it has a potential for breaking everything, but for pipelines files, no strong objection.

Leaving it as is for now to be consistend with the rest.